lib/pusher_channels_flutter.dart

```diff
@@ -195,12 +195,13 @@ class PusherChannelsFlutter {
         return Future.value(null);
       case 'onMemberAdded':
         var member = PusherMember(user["userId"], user["userInfo"]);
+        channels[channelName]?.members[member.userId] = member;
         onMemberAdded?.call(channelName!, member);
-        channels[channelName]?.members.add(member);
         channels[channelName]?.onMemberAdded?.call(member);
         return Future.value(null);
       case 'onMemberRemoved':
         var member = PusherMember(user["userId"], user["userInfo"]);
+        channels[channelName]?.members.remove(member.userId);
```

Comment:
Does here need a null check ? before remove method?

Reply:
Yes either that or a !, but that could theoretically cause an exception if the channel could not be found.